### ISSUE TITLE
extend latex2page to include equation numbering

### ISSUE THREAD

**Issue by kirbybrianj** [dsavransky/grading#14]
**Is your feature request related to a problem? Please describe.**
n/a

**Describe the solution you'd like**
HR32 says she wants latex2page to be expanded include equation numbering

**Additional context**
n/a

**Comment by dsavransky** [comment]
Ok.  I think I have a robust solution for this (but it is very, very ugly).  Basic idea:

1. Preflight - scan through LaTeX source and find all labeled equation environments (could alternatively pick up all numbered eqs, but if the point here is cross-referencing, then why number unreferenced eqs?).  Could maybe make this a user toggle (number all vs. only referenced).  In any case, scan through all equation, eqnarray, align (other?) envs and make dictionary of labels vs eq number.
2. In the pandoc-generated HTML, at the same time as we're replacing the codecogs url with the canvas url, for each equation to be numbered, update as follows:
```
 <span style="margin: 1ex auto; display: table; text-align: center; width: 100%; vertical-align: middle;">
 <img class="equation_image" style="display: block; text-align: center; margin: 0.5rem auto;" [orig eq. content] />
 <span style="display: table-cell; text-align: left; vertical-align: middle;">([equation nmber])</span>
 </span>
```
3. Update any references elsewhere in the HTML to the equation number instead of the label name (just a global replace like the others already in use).

Caveats: block math environments (eqnarray and align) can only have a single link as they are going to appear as a single image in the rendered html.  The internal dict should basically just map all relevant eq. numbers to a single reference.  I don't think this will be a major issue, but if people hate this, then just don't use these envs for eqs you want to reference in the text.

**Comment by dsavransky** [comment]
Addressed by v2.9.0.
